env-paths.js

```diff
@@ -0,0 +1,63 @@
+// Taken from https://github.com/sindresorhus/env-paths/
+
+const path = require('path');
+const os = require('os');
+
+const homedir = require('user-home');
+const tmpdir = os.tmpdir();
+const env = process.env;
+
+const macos = function (name) {
+	const library = path.join(homedir, 'Library');
+
+	return {
+		data: path.join(library, 'Application Support', name),
+		config: path.join(library, 'Preferences', name),
+		cache: path.join(library, 'Caches', name),
+		log: path.join(library, 'Logs', name),
+		temp: path.join(tmpdir, name)
+	};
+};
+
+const windows = function (name) {
+	const appData = env.LOCALAPPDATA || path.join(homedir, 'AppData', 'Local');
+
+	return {
+		// data/config/cache/log are invented by me as Windows isn't opinionated about this
```

Comment:
There isn't a point in ripping this module verbatim.  The usage just plucks the `.cache` property so the API should be reduced to that use case.